tasks/bamboo.yml

```diff
@@ -3,25 +3,12 @@
   set_fact:
     bamboo_master_binary_folder: "{{ bamboo_master_application_folder }}/atlassian-bamboo-{{ bamboo_master.version }}"
 
-- name: "Install JDK"
-  yum:
-    name: "java-{{ openjdk_version }}-openjdk"
-    state: installed
-
-- name: Create app and data folders
-  file:
-    name: "{{ item }}"
-    state: directory
-  with_items:
-    - "{{ bamboo_master_application_folder }}"
-    - "{{ bamboo_master_data_folder }}"
-
 - name: Add local user
   user:
     name: "{{ bamboo_master_user }}"
     home: "{{ bamboo_master_data_folder }}"
 
-- name: Set permissions for app and data folders
+- name: Create app and data folders
```

Comment:
Can you move this above task "Add local user"?

Reply:
Yep, it is moved.